neon_intrinsics/advsimd.rst

```diff
@@ -141,12 +141,16 @@ Document history
 +------------+-------------------------+-------------------------+
 |F           |30 May 2020              |Version ACLE Q2 2020     |
 +------------+-------------------------+-------------------------+
-|G           |30 October 2020          |Version ACLE Q2 2020     |
+|G           |30 October 2020          |Version ACLE Q3 2020     |
 +------------+-------------------------+-------------------------+
-|H           | |date-of-issue|         | |release|               |
+|H           |02 July 2021             |Version ACLE Q2 2021     |
 +------------+-------------------------+-------------------------+
 
 
+Changes between ACLE Q3 2020 and ACLE Q2 2021
```

Comment:
```suggestion
Changes between next release  and 2021Q2
```

Reply:
BTW, I like this line _a lot_, much more than the table. I might steal the idea from you :)

Reply:
```suggestion
Changes between next release and 2021Q2
```

Reply:
@MattPD - I am unable to apply this change I have requested via the github interface. Would you mind pushing another commit with the fixes? I don't know how to make it work otherwise... Apologies for the extra step.

Reply:
Updated just now; let me know if this looks good.